src/hs1.jl

```diff
@@ -0,0 +1,37 @@
+# Hock and Schittkowski problem number 1.
+#
+#   Source:
+#   Problem 1 in
+#   W. Hock and K. Schittkowski,
+#   Test examples for nonlinear programming codes,
+#   Lectures Notes in Economics and Mathematical Systems 187,
+#   Springer Verlag, Heidelberg, 1981.
+#
+#   classification QQR2-AN-2-1
```

Comment:
The classification strings are not the same as those in CUTEst. You can find the right ones in the corresponding SIF file. Sorry I didn't check that earlier.